### PR TITLE
Issue59/plugin status to frontend

### PR DESCRIPTION
**Pull request by mateuszkrasucki**
This is how it looks at the moment:

status_iryo

### CODE REVIEW

File: frontend/shared/containers/status/index.js [hunk 1]
```diff
@@ -1,10 +1,114 @@
...
+    componentDidMount() {
+        this.props.loadStatus()
+        this.interval = setInterval(this.props.loadStatus, 5000);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (!nextProps.status && this.props.status) {
+            this.props.loadStatus()
+        }
+    }
```
Comment (dominikznidar) [inline frontend/shared/containers/status/index.js]: Is this meant to trigger the initial load?

Comment (dominikznidar) [reply frontend/shared/containers/status/index.js]: If so you can just call it in `constructor`

Comment (mateuszkrasucki) [reply frontend/shared/containers/status/index.js]: DidMount does it, this is scheme I made all over the place that is originally meant for situations in which some operation triggers removing specific state from redux so when component that needs it is updated, it's refetched from backend. In this case it indeed doesn't need to be there. Will remove.

**Comment by mateuszkrasucki** [comment]
merged

**Pull request closed without merging**